confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/internal/input/ConfluenceInputFilterStream.java

```diff
@@ -1199,6 +1296,60 @@ private void readPageComment(Long pageId, String spaceKey, ConfluenceFilter prox
         proxyFilter.endWikiObject(COMMENTS_CLASSNAME, commentParameters);
     }
 
+    /**
+     * @since 9.24.0
+     */
+    private void storeBlogDescriptorPage(ConfluenceFilter proxyFilter) throws FilterException
```

Comment:
I would not use the term "store" as you have no idea at this level if this data is going to be stored, or converted to something else.